src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/CurrentUserProfileTest.java

```diff
@@ -0,0 +1,74 @@
+package com.cmput301f21t09.budgetprojectname;
+
+import org.junit.Rule;
+
+import android.app.Activity;
+import android.widget.EditText;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.robotium.solo.Solo;
+
+
+/**
+ * Test class for currentUserProfileFragment
+ */
+public class CurrentUserProfileTest {
+    private Solo solo;
+    @Rule
+    public ActivityTestRule<MainActivity> rule =
+            new ActivityTestRule<>(MainActivity.class, true, true);
+
+    /**
+     * Runs before all tests and creates solo instance.
+     *
+     * @throws Exception
+     */
+    @Before
+    public void setUp() throws Exception {
+        solo = new Solo(InstrumentationRegistry.getInstrumentation(), rule.getActivity());
+    }
+
+    /**
+     * Closes the activity after each test
+     *
+     * @throws Exception
+     */
+    @After
+    public void tearDown() throws Exception {
+        solo.finishOpenedActivities();
+    }
+
+    /**
+     * Gets the Activity
+     *
+     * @throws Exception
+     */
+    @Test
+    public void start() throws Exception {
+        Activity activity = rule.getActivity();
+    }
+
+    void navigateToPersonalProfile() {
```

Comment:
missing javadoc for this function. Also is the function supposed to be public or private?

Reply:
private and yes will add